sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/EnterCodeViewModel.kt

```diff
@@ -91,10 +91,12 @@ class EnterCodeViewModel @Inject constructor(
     private suspend fun authFlow(
         pastedCode: String,
     ) {
-        val userSignedIn = syncAccountRepository.isSignedIn()
+        val primaryKey = syncAccountRepository.getAccountInfo().primaryKey
         when (val result = syncAccountRepository.processCode(pastedCode)) {
             is Result.Success -> {
-                val commandSuccess = if (userSignedIn) {
+                val postProcessCodePK = syncAccountRepository.getAccountInfo().primaryKey
```

Comment:
could `postProcessCodePK` be `blank`? I see we check for blanks on the previous PK… what would it mean if the new one is `blank` if that’s possible?

Reply:
It's not possible inside the success branch, since success means user joined an account, so that means they have a primary key.